### PR TITLE
[bitnami/harbor] Add Harbor Helm Chart tests

### PR DESCRIPTION
**Pull request by FraPazGal**
**Description of the change**

**Test results**

[9 consecutive executions without issues.](https://github.com/FraPazGal/charts/actions/runs/2059449857)

Samples

- [GKE](https://cp.production.k.bromelia.bitnami.net/v1/execution-graphs/5d510d67-bd97-4d9a-b7d6-24202d0743d5/report) - PASS
- [AKS](https://cp.production.k.bromelia.bitnami.net/v1/execution-graphs/0b8a85a9-4cda-414e-94f3-e8dca89855eb/report) - PASS
- [TKG](https://cp.production.k.bromelia.bitnami.net/v1/execution-graphs/d8e5f55b-5f86-4db7-8e99-54be77eace6e/report) - PASS

Adding the tests for the Thanis Helm chart. 

**Benefits**

When someone makes a change in the Helm Chart, they will be able to check if their change has broken the Helm chart by running these automated tests.

**Possible drawbacks**

With UI tests, there is always a possibility for flakiness. 

**Checklist**

- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/harbor/cypress/cypress/integration/harbor_spec.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import {
+  random,
+} from './utils'
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('div', 'Invalid user name or password.').should('not.exist');
+  cy.contains('button','admin').click();
+  cy.contains('a', 'Log Out').click();
+  cy.get('.login-group');
```
Comment (superaleks) [inline .vib/harbor/cypress/cypress/integration/harbor_spec.js:11]: What does this do after getting?

File: .vib/harbor/cypress/cypress/integration/harbor_spec.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+})
+
+it('allows launching a vulnerability scan', () => {
+  const IMAGE_SCANNER = 'Trivy';
+  cy.login();
+  cy.visit('/harbor/interrogation-services/scanners');
+  cy.contains('.datagrid-table',IMAGE_SCANNER);
+  cy.contains('a', 'Vulnerability').click();
+  cy.contains('button','SCAN NOW').click();
+  cy.contains('.alert-item','Trigger scan all successfully!').click();
```
Comment (superaleks) [inline .vib/harbor/cypress/cypress/integration/harbor_spec.js:44]: Why is there a click on the alert? Does something relevant happen afterwards?

File: .vib/harbor/cypress/cypress/integration/harbor_spec.js (hunk 1 above)
Comment (FraPazGal) [reply .vib/harbor/cypress/cypress/integration/harbor_spec.js:11]: I'm using this as a check to verify the logout was correct and it redirected you to the login page. I updated the element for it to be clearer.

File: .vib/harbor/cypress/cypress/integration/harbor_spec.js (hunk 2 above)
Comment (FraPazGal) [reply .vib/harbor/cypress/cypress/integration/harbor_spec.js:44]: It does not, this was a typo. Thanks for pointing it out!

File: .vib/harbor/cypress/cypress/integration/harbor_spec.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (FraPazGal) [reply .vib/harbor/cypress/cypress/integration/harbor_spec.js:11]: You are right, modified class-based selectors whenever possible.

**Review by superaleks — DISMISSED** [review]
LGTM!

**Review by joancafom — APPROVED** [review]
LGTM

**Pull request merged**